streaming/java/streaming-api/src/main/java/org/ray/streaming/python/descriptor/Descriptor.java

```diff
@@ -0,0 +1,14 @@
+package org.ray.streaming.python.descriptor;
+
+/**
+ * Descriptor is used to describe python-related function/operator/partition.
+ */
+public interface Descriptor {
```

Comment:
This name sounds unclear to me. Because it's neither related to Python, nor related to function/operator/partition.

Reply:
Removed Descriptor and renamed `DescriptorFunction` to `PythonFunction`, `DescriptorPartition` to `PythonPartition`